### ISSUE TITLE
Detail View - Intertexts: correct citations

### ISSUE THREAD

**Issue by babslgam** [karl-kraus/wpn-static#88]
we now have tests for short and detail infos for persons and quotes/intertexts. some are failing and i would need some input, because i'm not sure anymore if its solved correctly on production:

- dot needed after ellipsis or not?
  - https://karl-kraus.github.io/wpn-static/absatz_24.html#DWquote0637_DWbibl03588
  - https://karl-kraus.github.io/wpn-static/absatz_45.html#DWquote1106_DWbibl03989
  - https://karl-kraus.github.io/wpn-static/absatz_63.html#DWquote1714_DWbibl03355

- final dot needed here?
  - https://karl-kraus.github.io/wpn-static/absatz_36.html#DWquote0858_DWbibl03979
  
- duplicated quotetype on production 
  - https://karl-kraus.github.io/wpn-static-test-reports/#?testId=c8ce496154f64b24914a-1d65afa982a3d87db548
  - https://karl-kraus.github.io/wpn-static-test-reports/#?testId=c8ce496154f64b24914a-748320324825b0028313
  - https://karl-kraus.github.io/wpn-static-test-reports/#?testId=c8ce496154f64b24914a-ee9df567e2655b1ac0d3
  
- data fix needed for DWquote1093 / source DWbibl02870 with ref DWbibl03235, references should not point to commented-out data.

- dot after issue number?
  - https://karl-kraus.github.io/wpn-static/absatz_61.html?#DWquote1645_DWbibl03069
  
- dot after paper title in these cases?
  - https://karl-kraus.github.io/wpn-static/absatz_62.html?#DWquote1674_DWbibl03083
  - https://karl-kraus.github.io/wpn-static/absatz_62.html?#DWquote1677_DWbibl03086

**Comment by BOberreither** [comment]
@babslgam 
> dot needed after ellipsis or not?
No, in none of these cases, so 1, 2 are correct, dot in 3 can be removed.

> final dot needed here?
Yes.

> duplicated quotetype on production
Are these all of the cases like this? Then we could decide anew (I don't know the ratio from back then any more, at least not exactly. Then we could turn the link from "Zit. in: Siehe z. B.: " to "Zit. z. B. in: "

> dot after issue number?
Yes.

> dot after paper title in these cases?
Yes.

**Comment by babslgam** [comment]
re: duplicated quotetype on production
they seem to be the only cases where the text (which is based on quote having type "exemp") seems to be duplicated before the link text. since the links in these cases have sub type specific, i think that the current versions are already correct, but please check again:

https://karl-kraus.github.io/wpn-static/absatz_42.html?#DWquote0971_DWbibl03909
https://karl-kraus.github.io/wpn-static/absatz_42.html?#DWquote2032_DWbibl03914
https://karl-kraus.github.io/wpn-static/absatz_64.html?#DWquote1851_DWbibl03402

(so in short: i think "Siehe z. B.:" has just been unnecessarily duplicated before.)

**Comment by BOberreither** [comment]
@babslgam Phew - we (ahem: I) sure loved to make it complicated back then; but I think it is alright now.
